saritasa_invocations/k8s.py

```diff
@@ -68,16 +67,15 @@ def get_current_env_config_from_context(
     config = _config.Config.from_context(context)
     for env in config.k8s_configs.values():
         if (
-            env.cluster == current_cluster
-            and current_namespace in env.namespace
-        ):
+            env.context or env.proxy
```

Comment:
Please make separate variables instead of multiline IF-s, example:
```python
context_matches = (env.context or env.proxy) == current_context
known_namespace = current_namespace in env.namespace
if context_matches and known_namespace:
    ...
```